frontend/src/components/Schedule/Schedule.tsx

```diff
@@ -20,9 +39,43 @@ const Schedule = () => {
   const defaultEnd = new Date(defaultStart.getTime() + 28699999);
 
   const [curStart, setCurStart] = useState(defaultStart);
-  const [events, setEvents] = useState(tempEvents);
+  const [events, setEvents] = useState<CalEvent[]>([]);
+  const [drivers, setDrivers] = useState<Driver[]>([]);
   const [viewState, setviewState] = useState(false);
 
+  useEffect(() => {
+    fetch('/rides')
+      .then((res) => res.json())
+      .then(({ data }) => {
+        console.log(data)
+        setEvents(data
+          .filter((ride: any) => ride.type !== "unscheduled" && ride.driver !== undefined)
+          .map((ride: any) => ({
+            id: ride.id,
+            title:
+              "" + ride.startLocation.name + " to " + ride.endLocation.name + "\n" +
+              "Rider: " + ride.rider.firstName + " " + ride.rider.lastName + "\n" +
+              "Driver: " + ride.driver.firstName + " " + ride.driver.lastName,
+            start: new Date(ride.startTime.toString()),
+            end: new Date(ride.endTime.toString()),
+            resourceId: ride.driver.id
+          })),
+        )
+      }
+      );
```

Comment:
these end brackets/parens can be cleaned up, not sure if you have eslint set up but after formatting this is what it looks like for me
![image](https://user-images.githubusercontent.com/20343975/101293018-171edd00-37e1-11eb-9660-b27ec039b84e.png)